### ISSUE TITLE
Indirect branches are not resolved

### ISSUE THREAD

**Issue by mrphrazer** [Vector35/binaryninja-api#3010]
For version `3.0.3285-dev` ( and older ones),  binja does not resolve indirect branch targets for the attached binary _although_ it correctly identifies the jump table entries. I don't see any reason why it shouldn't be able to determine those targets. The branch targets are also not recognized as code.

Code:

```
080494fa  8b55d8             mov     edx, dword [ebp-0x28]
080494fd  8b0495cc2d0b08     mov     eax, dword [edx*4+0x80b2dcc]
08049504  ffe0               jmp     eax
```
Jump table:

```
080b2dcc  uint32_t jump_table_80b2dcc[0xa] = 
080b2dcc  {
080b2dcc      [0x0] =  0x080496d9
080b2dd0      [0x1] =  0x080496d9
080b2dd4      [0x2] =  0x08049506
080b2dd8      [0x3] =  0x08049515
080b2ddc      [0x4] =  0x080496d9
080b2de0      [0x5] =  0x080496d9
080b2de4      [0x6] =  0x08049610
080b2de8      [0x7] =  0x08049645
080b2dec      [0x8] =  0x08049677
080b2df0      [0x9] =  0x080496a9
080b2df4  }
```

[sample.zip (pw: infected)](https://github.com/Vector35/binaryninja-api/files/8233421/sample.zip)

**Comment by psifertex** [comment]
The jump table isn't actually identified out of the box -- I assume you used the workaround described below to create the jump table and then used Undo? So I think there's another minor bug here that it's not undoing the creation of the jump table. 

As for the original issue, it's simply a matter of value-set analysis not working on stack variables. We don't currently have an alias analysis that would let us make safe assumptions about stack variables. 

A quick work-around is to select the MLIL VAR edx_2, right-click, select `set user variable value` and enter a `signed range value` of `{0:9:1}`. 
![Screen Shot 2022-03-11 at 11 08 56 AM](https://user-images.githubusercontent.com/140215/157904587-1a57f04f-cdeb-4e85-9e8e-bc148e6c18ea.png)

**Comment by mrphrazer** [comment]
No, I didn't create anything manually. If I do a fresh analysis, go to the code location and double click on the address `0x80b2dcc`, it's defined as above. It's also not a matter of symbols, it works the same way (automatically) on the stripped binary

**Comment by psifertex** [comment]
I don't see that same behavior. Are you sure there isn't something else going on?
![Screen Shot 2022-03-11 at 11 26 24 AM](https://user-images.githubusercontent.com/140215/157907575-20067660-f083-49cf-9916-ffb3d421281e.png)

**Comment by bpotchik** [comment]
Fixed in `3.0.3292-dev`.
